ts/features/ingress/store/selectors/index.ts

```diff
@@ -0,0 +1,6 @@
+import { GlobalState } from "../../../../store/reducers/types";
+
+const ingressScreenSelector = (state: GlobalState) => state.ingress;
+
+export const isBlockingScreenSelector = (state: GlobalState) =>
+  ingressScreenSelector(state).isBlockingScreen;
```

Comment:
What about accessing it directly 
```suggestion
  state.ingress.isBlockingScreen;
```
without the need to pass inside another function (previous selector).

For me is much more clean and efficient.

Reply:
Since I'll move the `ingress` reducer under `features` I'll adapt the selector like you suggest but with the updated path. Thanks!